modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -224,3 +243,6 @@ familyName: "Mukai"
 givenName: "Kyung-won"
 familyName: "Park"
 ----
+
+API Designer simply ignores the header and uses the examples. However, if you planned to validate your specification with other RAML processors, these two fragments would generate errors or warnings.
```

Comment:
> with other RAML processors

I would add “any” before “other”

> would generate errors or warnings

I would replace “would” by “should” here. I would replace “or warnings” by “as per RAML 1.0 specification”.

Reply:
@jstoiko , I've changed the paragraph to this:
"API Designer simply ignores the header and uses the examples. However, if you validate a specification with any other RAML processors, NamedExample fragments that define only one example should generate errors, in accord with the RAML 1.0 specification."